src/lib/support/BufBound.h

```diff
@@ -68,6 +69,7 @@ class BufBound
         }
         return mWritten;
     }
+    size_t Put8(uint8_t c) { return Put(c); }
```

Comment:
I wonder if BufBound should return itself like BufReader does.  returning bytes written is generally uninteresting to clients

Reply:
+1 to that. I did not want to change the interface more, however would absolutely support a PR changing things to return *this. May make other code cleaner as a result as well.